src/models/differentiable/evotree_model.jl

```diff
@@ -0,0 +1,147 @@
+using DataFrames
+using SliceMap
+using EvoTrees
+using MLJBase
+
+"""
+This type provides a basic interface to gradient-boosted tree models from the MLJ library.
+However, this is not be the final version of the interface: full support for EvoTrees has not been implemented yet and the `logits` and `probs` methods will be changed in the process of doing that if needed.
+"""
+
+"""
+    EvoTreeModel <: AbstractMLJModel
+
+Constructor for gradient-boosted decision trees from the EvoTrees.jl library. 
+"""
+struct EvoTreeModel <: AbstractDifferentiableModel
+    model::Any
+    likelihood::Symbol
+    function EvoTreeModel(model, likelihood)
+        if likelihood ∈ [:classification_binary, :classification_multi]
+            new(model, likelihood)
+        else
+            throw(
+                ArgumentError(
+                    "`type` should be in `[:classification_binary, :classification_multi]1.
+                    Support for regressors has not been implemented yet.`"
```

Comment:
I suppose there's a typo (this "1" at the end of the 25th line)

Reply:
Thanks, fixed.